metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/WorkflowPostProcessor.java

```diff
@@ -91,12 +98,15 @@ void performPluginPostProcessing(AbstractExecutablePlugin<?> plugin, String data
   /**
    * Performs post-processing for indexing plugins
    *
-   * @param indexPlugin the index plugin
-   * @param datasetId the dataset id
-   * @throws DpsException if communication with ecloud dps failed
+   * @param indexPlugin The index plugin
+   * @param datasetId   The dataset id
+   * @throws DpsException                If communication with e-cloud dps failed
```

Comment:
I don't think we have this spacing in our code style(I'm not sure though).
Are you using the correct one?